### PR TITLE
Add and symlink sock_link_extra files

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd

### CODE REVIEW

File: lagunita/cms/templates/widgets/sock_links_extra.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<%!
+from django.conf import settings
+from django.utils.translation import ugettext as _
+%>
+
+<%def name="get_sock_links_extra()">
+    <%
+        links_extra = [
+            {
+                'href': 'https://stanfordonline.zendesk.com/hc',
```
Comment (stvstnfrd) [inline lagunita/cms/templates/widgets/sock_links_extra.html]: Isn't there a variable for this? like `mktg_link('FAQ')` or some such?

File: lagunita/cms/templates/widgets/sock_links_extra.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+                'sr_mouseover_text': _('Find answers in the FAQ or ask a new question.'),
+                'text': _('Help Center'),
+                'condition': True
```
Comment (stvstnfrd) [inline lagunita/cms/templates/widgets/sock_links_extra.html]: Trailing comma?

File: lagunita/cms/templates/widgets/sock_links_extra.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+            }
```
Comment (stvstnfrd) [inline lagunita/cms/templates/widgets/sock_links_extra.html]: Trailing comma?

**Comment by caesar2164** [comment]
Ok, @stvstnfrd, with the new commit on https://github.com/Stanford-Online/edx-platform/pull/768 this now works in my devstack.

File: lagunita/cms/templates/widgets/sock_links_extra.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<%page expression_filter="h" />
+<%!
+from django.conf import settings
```
Comment (stvstnfrd) [inline lagunita/cms/templates/widgets/sock_links_extra.html]: Are we using `settings`? Do we need this?

Comment (caesar2164) [reply lagunita/cms/templates/widgets/sock_links_extra.html]: We don't! This is a remnant from "copy-paste".

**Comment by caesar2164** [comment]
@stvstnfrd retested in devstack, looks good.

**Review by stvstnfrd — APPROVED** [review]

**Pull request merged**